lib/__tests__/profile.test.js

```diff
@@ -0,0 +1,341 @@
+  let encryptedAk = await Crypto.encryptAk(cryptoKeys.privateKey, ak, cryptoKeys.publicKey)
+  eak = new EAKInfo(encryptedAk, clientId, cryptoKeys.publicKey, clientId, signingKeys.publicKey)
+
+  data1B = await randomRecordData(1)
+  data500B = await randomRecordData(500)
+  data250KB = await randomRecordData(250000)
+  data500KB = await randomRecordData(500000)
+  data1MB = await randomRecordData(1000000)
+  data2MB = await randomRecordData(2000000)
+  data4MB = await randomRecordData(4000000)
+
+  done()
+})
+
+describe('Profile', () => {
+  it('tests record sizes', async (done) => {
+    rec1B = await client.encrypt(recordType, data1B, eak)
+    rec500B = await client.encrypt(recordType, data500B, eak)
+    rec250KB = await client.encrypt(recordType, data250KB, eak)
+    rec500KB = await client.encrypt(recordType, data500KB, eak)
+    rec1MB = await client.encrypt(recordType, data1MB, eak)
+    rec2MB = await client.encrypt(recordType, data2MB, eak)
+    rec4MB = await client.encrypt(recordType, data4MB, eak)
+
+    signed1B = await client.sign(data1B)
+    signed500B = await client.sign(data500B)
+    signed250KB = await client.sign(data250KB)
+    signed500KB = await client.sign(data500KB)
+    signed1MB = await client.sign(data1MB)
+    signed2MB = await client.sign(data2MB)
+    signed4MB = await client.sign(data4MB)
+
+    printSizes()
+
+    done()
+  })
+
+  it('tests 1B', async (done) => {
+    let enc, dec, sign, verify
+
+    do {
+      let start = Date.now()
+      let encrypted = await client.encrypt(recordType, data1B, eak)
+      let pause = Date.now()
+      await client.decrypt(encrypted, eak)
+      dec = Date.now() - pause
+      enc = pause - start
+    } while (false)
+
+    do {
+      let start = Date.now()
+      let signature = await client.sign(data1B)
+      let pause = Date.now()
+      await client.verify(new SignedDocument(data1B, signature), signingKeys.publicKey)
+      verify = Date.now() - pause
+      sign = pause - start
+    } while (false)
+
+    console.log(
+      util.format(
+        'Size 1B; Encrypt %dms; Decrypt %dms; Sign %dms; Verify %dms;',
+        enc, dec, sign, verify
+      )
+    )
+
+    done()
+  })
+
+  it('tests 500B', async (done) => {
+    let enc, dec, sign, verify
+
+    do {
+      let start = Date.now()
+      let encrypted = await client.encrypt(recordType, data500B, eak)
+      let pause = Date.now()
+      await client.decrypt(encrypted, eak)
+      dec = Date.now() - pause
+      enc = pause - start
+    } while (false)
+
+    do {
+      let start = Date.now()
+      let signature = await client.sign(data500B)
+      let pause = Date.now()
+      await client.verify(new SignedDocument(data500B, signature), signingKeys.publicKey)
+      verify = Date.now() - pause
+      sign = pause - start
+    } while (false)
+
+    console.log(
+      util.format(
+        'Size 500B; Encrypt %dms; Decrypt %dms; Sign %dms; Verify %dms;',
+        enc, dec, sign, verify
+      )
+    )
```

Comment:
Seems like you could refactor to a single function used by each test, passing in the `dataN` and the string describing the size (e.g. "500 B").